flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/source/enumerator/TestContinuousSplitPlannerImplStartStrategy.java

```diff
@@ -165,11 +165,11 @@ public void testForSpecificSnapshotTimestampStrategySnapshot2() throws IOExcepti
             .startSnapshotTimestamp(1L)
             .build();
 
-    // empty table
     assertThatThrownBy(
             () ->
                 ContinuousSplitPlannerImpl.startSnapshot(
                     TABLE_RESOURCE.table(), scanContextInvalidSnapshotTimestamp))
+        .as("empty table")
```

Comment:
I feel like this rather adds confusion than helping if the assertion ever fails. Aren't we just providing an invalid snapshot timestamp and thus not getting any snapshot back? I would just remove `.as("empty table")` here and in the other assertions

Reply:
Thanks for checking this. Yes, agree with your comment. Let me fix this and other part.